src/Vehicle/Vehicle.cc

```diff
@@ -96,6 +96,10 @@ const char* Vehicle::_distanceToGCSFactName =       "distanceToGCS";
 const char* Vehicle::_hobbsFactName =               "hobbs";
 const char* Vehicle::_throttlePctFactName =         "throttlePct";
 
+//const char*  Vehicle::_fuelLevelFactName =          "fuelLevel";
+const char*  Vehicle::_engineRPMFactName =          "engineRPM";
+const char*  Vehicle::_rotorRPMFactName =           "rotorRPM";
```

Comment:
These changes are not included in the documentation

Reply:
added in doc